dbatools.psd1

```diff
@@ -11,7 +11,7 @@
     RootModule             = 'dbatools.psm1'
 
     # Version number of this module.
-    ModuleVersion          = '1.0.134'
+    ModuleVersion          = '1.0.135'
```

Comment:
This should be reverted or the file reset to be removed from the PR.

Reply:
D’oh 🤦

Reply:
fixed in latest commit